### PR TITLE
Assisted in add positions

### PR DESCRIPTION
**Pull request by ANGIECATHERINEP**
Assisted in add positions

### CODE REVIEW

File: src/pages/privileges/outlets/positions/add-position/config/addPosition.config.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        isActive: false,
+      },
+      {
+        path: "/privileges/positions",
+        label: "Cargos",
+        id: "/privileges/positions",
+        isActive: false,
+      },
+      {
+        path: "/privileges/positions/add-positions",
```
Comment (LinconOssa) [inline src/pages/privileges/outlets/positions/add-position/config/addPosition.config.tsx:51]: The route is wrong here, it would be add-position

**Review by LinconOssa — COMMENTED** [review]
In the previous activity of the table, the path must be corrected so that the assisted can be seen:
image

File: src/pages/privileges/outlets/positions/add-position/config/addPosition.config.tsx (hunk 1 above)
Comment (ANGIECATHERINEP) [reply src/pages/privileges/outlets/positions/add-position/config/addPosition.config.tsx]: resolved

**Review by LinconOssa — APPROVED** [review]
Meets requirements

**Review by carin91 — APPROVED** [review]

**Pull request merged**